src/test/java/io/github/bobocodebreskul/context/registry/SimpleBeanDefinitionRegistryTest.java

```diff
@@ -245,9 +249,13 @@ void given_BeanName_When_RemoveBeanDefinition_Then_RegistryNotContainBean() {
         .as("Bean definition with name TEST_BEAN_NAME_1 should not be null")
         .isNotNull();
     simpleBeanDefinitionRegistry.removeBeanDefinition(TEST_BEAN_NAME_1);
-    assertThat(simpleBeanDefinitionRegistry.getBeanDefinition(TEST_BEAN_NAME_1))
-        .as("Bean definition with name TEST_BEAN_NAME_1 should be null after removing")
-        .isNull();
+
+    Exception actualException = catchException(() ->
+        simpleBeanDefinitionRegistry.getBeanDefinition(TEST_BEAN_NAME_1));
+    assertThat(actualException)
+        .isInstanceOf(NoSuchBeanDefinitionException.class)
+        .hasMessage(BEAN_DEFINITION_FOR_NAME_NOT_FOUND.formatted(TEST_BEAN_NAME_1));
```

Comment:
This check is done in `given_BeanName_When_BeanDefinitionNotExists_Then_ThrowNoSuchBeanDefinitionException` so no need to check it here.

Reply:
I tested that this definition was deleted